xamarin/Xamarin.Forms
# [Android] Fix SwipeItems dispose when closing SwipeView

### Description of Change ###

On Android, doing swipe we could create a new SwipeItems view, so closing the SwipeView and removing / disposing the view, we did not obtain the expected result. This PR validates the dispose of the SwipeItems view.

### Issues Resolved ### 

- fixes #9646
- fixes #9647 
- fixes #9735 

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

#### Before
![issue9646-before](https://user-images.githubusercontent.com/6755973/75358925-ea35b500-58b3-11ea-8292-668967932d10.gif)

#### After
![issue9646-after](https://user-images.githubusercontent.com/6755973/75358928-eace4b80-58b3-11ea-9ea9-b05cacb8e989.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 9646. Open a SwipeView and scroll. Verify that there are no unexpected SwipeViews open, etc.

### PR Checklist ###
